redash/tasks/worker.py

```diff
@@ -3,7 +3,8 @@
 import signal
 import time
 from redash import statsd_client
-from rq import Worker as BaseWorker, Queue as BaseQueue, get_current_job
+from rq import Queue as BaseQueue, get_current_job
+from rq.worker import HerokuWorker as BaseWorker
```

Comment:
Let's import `HerokuWorker` as is and update the reference where we use `BaseWorker` otherwise it might be missed if it's buried in the import section. Also worth adding a comment on why we use it - similar to what you have in the commit comment.

Reply:
Updated